src/actions/secrets.js

```diff
@@ -166,13 +166,18 @@ export function createSecret(postData, namespace) {
         namespace: serviceAccount.metadata.namespace,
         secretName: postData.metadata.name
       });
+      dispatch({
+        type: 'CLEAR_SECRET_ERROR_NOTIFICATION'
+      });
       dispatch({
         type: 'SECRET_CREATE_SUCCESS'
       });
+      return false;
```

Comment:
wouldn't it be better to use `SECRET_CREATE_SUCCESS` and `SECRET_CREATE_FAILURE` to initialize the result variable and get the value from the reducer?

Reply:
Do you mean that in the reducer `errorMessage` should return false instead of the error? i'm a little confused as to how this would work as then the reducer wouldn't be returning the error?